corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```
@@ -302,3 +302,13 @@
 .gr-has-no-nested-questions {
   display: none;
 }
+
+.checkbox, .radio {
+  input[type="checkbox"], input[type="radio"] {
+    // This needs customization any time the font size in Web Apps is updated
```

Might be worth rewording so it passes the "grep test" if the variable gets updated in the future:
```suggestion
    // This needs customization any time @form-text-size is updated
```